userspace/libsinsp/metrics_collector.cpp

```diff
@@ -112,6 +116,8 @@ std::string metric_value_to_text(const metrics_v2& metric)
 	case METRIC_VALUE_TYPE_I:
 		value_text = std::to_string(metric.value.i);
 		break;
+	default:
```

Comment:
`ASSERT(false)` here too perhaps? 
Anyway, going to approve this one so that we can move on!

Reply:
Thanks yes we can add this in later.